lib/content-services/src/lib/common/services/rendition.services.spec.ts

```diff
@@ -0,0 +1,105 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TestBed } from '@angular/core/testing';
+import { AlfrescoApiService, LogService, TranslationService, ViewUtilService } from '@alfresco/adf-core';
+import { RenditionEntry, RenditionPaging, RenditionsApi } from '@alfresco/js-api';
+import { RenditionService } from '@alfresco/adf-content-services';
+
+const getRenditionEntry = (status: 'NOT_CREATED' | 'CREATED'): RenditionEntry => {
```

Comment:
In js-api there is reusable const for these strings. Could you please use that const instead of hardcoded strings in whole file? It would be better to avoid typo in future etc. 
<img width="697" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/7972fe1b-9866-4a94-8056-ad30a24e7b97">